fdbserver/LogSystemPeekCursor.actor.cpp

```diff
@@ -622,7 +660,7 @@ ILogSystem::MergedPeekCursor::MergedPeekCursor(
 	for (int i = 0; i < logServers.size(); i++) {
 		bool returnEmptyIfStopped =
 		    ((SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST && end != std::numeric_limits<Version>::max())
```

Comment:
already check `end != std::numeric_limits<Version>::max()` in the helper, this is redundant